helm/cluster-cloud-director/templates/cleanup-helmreleases-hook-job.yaml

```
@@ -124,3 +125,26 @@ spec:
             limits:
               memory: "256Mi"
               cpu: "100m"
+{{- if not .Values.connectivity.network.allowAllEgress }}
+---
+apiVersion: cilium.io/v2
+kind: CiliumNetworkPolicy
+metadata:
+  labels:
+    {{- include "labels.common" $ | nindent 4 }}
+  name: "allow-{{ include "resource.default.name" $ }}-cleanup-helmreleases-hook-kubernetes-talk-to-apiserver"
```

maybe just shorten it a bit? not really an issue, just personal preference - makes no difference either way

```suggestion
  name: "allow-{{ include "resource.default.name" $ }}-cleanup-helmreleases-hook-to-apiserver"
```

---

Applied in https://github.com/giantswarm/cluster-cloud-director/pull/248/commits/93124f2e16f09e0b82ebc85900d9b7513c1b29c3